.circleci/config.yml

```diff
@@ -65,15 +73,100 @@ workflows:
   ci:
     jobs:
       - bundle:
+          name: "bundle_ruby2-6_rails5-2"
```

Comment:
These jobs should be vertically integrated (please see https://github.com/samvera/om/pull/78#pullrequestreview-238616619)